spec/index.html

```diff
@@ -55,7 +55,8 @@ <h1>Options</h1>
       <tr><td>`--assets-dir`</td><td>`assetsDir`</td><td>Directory in which to place assets when using `--assets=external`. Defaults to "assets".</td></tr>
       <tr><td>`--lint-spec`</td><td>`lintSpec`</td><td>Enforce some style and correctness checks.</td></tr>
       <tr><td>`--error-formatter`</td><td></td><td>The <a href="https://eslint.org/docs/user-guide/formatters/">eslint formatter</a> to be used for printing warnings and errors when using `--verbose`. Either the name of a built-in eslint formatter or the package name of an installed eslint compatible formatter.</td></tr>
-      <tr><td>`--max-clause-depth N`</td><td></td><td>Warn when clauses exceed a nesting depth of N, and cause those clauses to be numbered by incrementing their parent clause's number rather than by nesting a new number within their parent clause.</td></tr>
+      <tr><td>`--max-clause-depth N`</td><td>`maxClauseDepth`</td><td>Warn when clauses exceed a nesting depth of N, and cause those clauses to be numbered by incrementing their parent clause's number rather than by nesting a new number within their parent clause. Default 6.</td></tr>
+      <tr><td>`--toc-depth N`</td><td>`tocDepth`</td><td>Maximum level of subclause to be emitted in table of contents. Default 3.</td></tr>
```

Comment:
Looks like this only applies to printed documents? If so this should be named something like `--printable-toc-depth`, or at least have that behavior documented here

Reply:
Thank you, parameter and documentation both updated.